### ISSUE TITLE
Previous reign information

### ISSUE THREAD

**Issue by MarcellaSCA** [drachenwald/insulae-draconis#26]
Path: /coronet/
A lot of the information on the lower half of the page seems to be from the previous reign, re: attendants and whatnot.

**Comment by IDWebminister** [comment]
Yup - where do we get the correct information from?

**Comment by MarcellaSCA** [comment]
That was a choice that Alexandre and Eularia made for their reign - it’s not a thing everyone does. You could probably remove it entirely.

**Comment by IDWebminister** [comment]
Ahh. That makes sense. I just assumed it was a standard page that got
tweaked at the start of a new reign.

So I could talk to the P&P about what they would like to see there.

Thanks!

Yours in Service,

Eadbald
Webminister, Insulae Draconis

**Comment by MarcellaSCA** [comment]
Yep! The ’normal' stuff goes as far as the Champions. If they want to include their household, they can, but there’s no standard set for that or what positions are included. Alexandre and Eularia wanted to showcase everyone who attended them during their reign on this page, as well, which is what the rest of the stuff is. But, really, it can be whatever the Coronets want for the duration of their reign.

**Comment by MarcellaSCA** [comment]
Sorry - Protectors, not Champions!

**Comment by MarcellaSCA** [comment]
Oh! And the About the Coronets part. That’s static at the bottom of the page.
